src/store/driveSlice.js

```diff
@@ -26,10 +26,10 @@ const driveSlice = createSlice({
     },
 
     requestHolonomicDrive(state, action) {
-      const { straight, holoLeft, holoTurnccw } = action.payload;
-      state.straight = straight;
+      const { holoStraight, holoLeft, holoTurnCCW } = action.payload;
```

Comment:
Remember to initialize `holoStraight` in the initial state defined at the top of this file